code/modules/mob/living/carbon/diona_base.dm

```diff
@@ -0,0 +1,548 @@
+//This function is for code that is shared by diona nymphs and gestalt
+/mob/living/carbon/var/stored_energy
```

Comment:
Are these necessary for other carbon mobs, or can you get away with giving these only to diona?